src/Orleans/Streams/Core/IStreamSubscriptionManagerAdmin.cs

```diff
@@ -8,6 +8,11 @@ namespace Orleans.Streams.Core
 {
     public interface IStreamSubscriptionManagerAdmin
     {
-        IStreamSubscriptionManager GetStreamSubscriptionManager(string name);
+        IStreamSubscriptionManager GetStreamSubscriptionManager(StreamSubscriptionManagerType managerType);
```

Comment:
I think the type should remain a string, as having it as a defined enum means we can't add new ones without code change.  Limits extensibility.
We can have defined string values for the natively supported types.